Clipboard.xs

```diff
@@ -468,6 +468,34 @@ PPCODE:
 		XSRETURN_NO;
 	}
 
+void
+UGet()
+PPCODE:
+    HANDLE myhandle;
+    if(OpenClipboard(NULL)) {
+	EXTEND(SP,1);
+	if(myhandle = GetClipboardData(CF_UNICODETEXT)) {
+	    /* here we decode UTF16-LE into UTF8, using perl API */
+	    wchar_t *wcmyh = (wchar_t*)myhandle;
```

Comment:
Shouldn't this be a GlobalLock(), like in the example at https://docs.microsoft.com/en-us/windows/win32/dataxchg/using-the-clipboard#pasting-information-from-the-clipboard  ?

Reply:
yes, I think you're correct...
thanks